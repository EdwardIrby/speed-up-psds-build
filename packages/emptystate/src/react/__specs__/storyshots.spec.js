import path from 'path'
import initStoryshots, {
  snapshotWithOptions
} from '@storybook/addon-storyshots'

jest.mock('@pluralsight/ps-design-system-storybook-addon-center')
jest.mock('@pluralsight/ps-design-system-storybook-addon-theme')

jest.mock('../resize-observer')

const createNodeMock = el => document.createElement('div')

initStoryshots({
  configPath: path.resolve(__dirname, '..', '..', '..', '.storybook'),
  test: snapshotWithOptions({ createNodeMock })
})
