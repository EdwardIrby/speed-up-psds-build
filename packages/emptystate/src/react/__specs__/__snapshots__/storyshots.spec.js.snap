// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots EmptyState combo 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  />
  <h1
    data-css-amk6wt=""
  >
    Filet mignon pork chop salami shank tail.
  </h1>
  <p
    data-css-hk53qc=""
  >
    Burgdoggen boudin shoulder, sirloin turducken landjaeger kielbasa brisket. Bacon venison meatball pork, alcatra capicola turducken. Salami shoulder strip steak filet mignon pork.
  </p>
  <div
    data-css-aelvpu=""
  >
    <button
      data-css-176v989=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        Do a Thing
      </span>
    </button>
  </div>
</div>
`;

exports[`Storyshots EmptyState in a small fixed size container 1`] = `
<div
  style={
    Object {
      "width": 400,
    }
  }
>
  <div
    data-css-g3x4fp=""
  >
    <div
      data-css-1gwvklx=""
    />
    <h1
      data-css-amk6wt=""
    >
      Filet mignon pork chop salami shank tail.
    </h1>
    <p
      data-css-hk53qc=""
    >
      Burgdoggen boudin shoulder, sirloin turducken landjaeger kielbasa brisket. Bacon venison meatball pork, alcatra capicola turducken. Salami shoulder strip steak filet mignon pork.
    </p>
    <div
      data-css-aelvpu=""
    >
      <button
        data-css-176v989=""
        disabled={false}
        style={Object {}}
      >
        <span
          data-css-15b13by=""
        >
          Do a Thing
        </span>
      </button>
    </div>
  </div>
</div>
`;

exports[`Storyshots EmptyState/actions with an input 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  />
  <h1
    data-css-amk6wt=""
  >
    Filet mignon pork chop salami shank tail.
  </h1>
  <p
    data-css-hk53qc=""
  >
    Burgdoggen boudin shoulder, sirloin turducken landjaeger kielbasa brisket. Bacon venison meatball pork, alcatra capicola turducken. Salami shoulder strip steak filet mignon pork.
  </p>
  <div
    data-css-aelvpu=""
  >
    <label
      data-css-w0yocm=""
    >
      <div
        data-css-1r967oi=""
      >
        <div
          data-css-ctre2o=""
        >
          <div
            data-css-1c5qy71=""
          >
            <input
              data-css-r9d4vy=""
              disabled={false}
              placeholder="Find Something"
              size="medium"
            />
          </div>
        </div>
        <div
          data-css-1os68q9=""
        >
          <div
            data-css-7dab2f=""
          >
            <svg
              aria-label="search icon"
              role="img"
              viewBox="0 0 24 24"
            >
              <path
                clipRule="evenodd"
                d="M21.353 20.648a.5.5 0 000-.708l-5.039-5.04A7.928 7.928 0 0018 10a7.948 7.948 0 00-2.344-5.656A7.942 7.942 0 0010 2a7.94 7.94 0 00-5.656 2.344A7.944 7.944 0 002 10a7.94 7.94 0 002.344 5.656A7.94 7.94 0 0010 18a7.926 7.926 0 004.9-1.685l5.04 5.04a.5.5 0 00.707 0l.706-.707zm-7.111-6.405A5.96 5.96 0 0110 16a5.959 5.959 0 01-4.242-1.757A5.958 5.958 0 014 10c0-1.602.624-3.109 1.758-4.242A5.955 5.955 0 0110 4c1.602 0 3.109.624 4.242 1.758A5.957 5.957 0 0116 10c0 1.603-.624 3.11-1.758 4.243z"
                fillRule="evenodd"
              />
            </svg>
          </div>
        </div>
      </div>
    </label>
  </div>
</div>
`;

exports[`Storyshots EmptyState/actions with buttons 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  />
  <h1
    data-css-amk6wt=""
  >
    Filet mignon pork chop salami shank tail.
  </h1>
  <p
    data-css-hk53qc=""
  >
    Burgdoggen boudin shoulder, sirloin turducken landjaeger kielbasa brisket. Bacon venison meatball pork, alcatra capicola turducken. Salami shoulder strip steak filet mignon pork.
  </p>
  <div
    data-css-aelvpu=""
  >
    <div
      style={
        Object {
          "display": "flex",
          "flexWrap": "wrap",
          "justifyContent": "center",
        }
      }
    >
      <div
        style={
          Object {
            "display": "inline-flex",
            "margin": "0 5px 12px",
          }
        }
      >
        <button
          data-css-1pp3uul=""
          disabled={false}
          style={Object {}}
        >
          <span
            data-css-15b13by=""
          >
            Asssessments
          </span>
        </button>
      </div>
      <div
        style={
          Object {
            "display": "inline-flex",
            "margin": "0 5px 12px",
          }
        }
      >
        <button
          data-css-1pp3uul=""
          disabled={false}
          style={Object {}}
        >
          <span
            data-css-15b13by=""
          >
            Guides
          </span>
        </button>
      </div>
      <div
        style={
          Object {
            "display": "inline-flex",
            "margin": "0 5px 12px",
          }
        }
      >
        <button
          data-css-1pp3uul=""
          disabled={false}
          style={Object {}}
        >
          <span
            data-css-15b13by=""
          >
            Interactive Courses
          </span>
        </button>
      </div>
      <div
        style={
          Object {
            "display": "inline-flex",
            "margin": "0 5px 12px",
          }
        }
      >
        <button
          data-css-1pp3uul=""
          disabled={false}
          style={Object {}}
        >
          <span
            data-css-15b13by=""
          >
            Paths
          </span>
        </button>
      </div>
      <div
        style={
          Object {
            "display": "inline-flex",
            "margin": "0 5px 12px",
          }
        }
      >
        <button
          data-css-1pp3uul=""
          disabled={false}
          style={Object {}}
        >
          <span
            data-css-15b13by=""
          >
            Projects
          </span>
        </button>
      </div>
      <div
        style={
          Object {
            "display": "inline-flex",
            "margin": "0 5px 12px",
          }
        }
      >
        <button
          data-css-1pp3uul=""
          disabled={false}
          style={Object {}}
        >
          <span
            data-css-15b13by=""
          >
            Video Courses
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots EmptyState/actions with tags 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  />
  <h1
    data-css-amk6wt=""
  >
    Filet mignon pork chop salami shank tail.
  </h1>
  <p
    data-css-hk53qc=""
  >
    Burgdoggen boudin shoulder, sirloin turducken landjaeger kielbasa brisket. Bacon venison meatball pork, alcatra capicola turducken. Salami shoulder strip steak filet mignon pork.
  </p>
  <div
    data-css-aelvpu=""
  >
    <div
      style={
        Object {
          "display": "flex",
          "flexWrap": "wrap",
          "justifyContent": "center",
        }
      }
    >
      <div
        style={
          Object {
            "display": "inline-flex",
            "margin": "0 5px 12px",
          }
        }
      >
        <div
          data-css-1lu7sfl=""
        >
          <div
            data-css-q4lcrt=""
          >
            <span
              data-css-1997kh1=""
            >
              Asssessments
            </span>
          </div>
        </div>
      </div>
      <div
        style={
          Object {
            "display": "inline-flex",
            "margin": "0 5px 12px",
          }
        }
      >
        <div
          data-css-1lu7sfl=""
        >
          <div
            data-css-q4lcrt=""
          >
            <span
              data-css-1997kh1=""
            >
              Guides
            </span>
          </div>
        </div>
      </div>
      <div
        style={
          Object {
            "display": "inline-flex",
            "margin": "0 5px 12px",
          }
        }
      >
        <div
          data-css-1lu7sfl=""
        >
          <div
            data-css-q4lcrt=""
          >
            <span
              data-css-1997kh1=""
            >
              Interactive Courses
            </span>
          </div>
        </div>
      </div>
      <div
        style={
          Object {
            "display": "inline-flex",
            "margin": "0 5px 12px",
          }
        }
      >
        <div
          data-css-1lu7sfl=""
        >
          <div
            data-css-q4lcrt=""
          >
            <span
              data-css-1997kh1=""
            >
              Paths
            </span>
          </div>
        </div>
      </div>
      <div
        style={
          Object {
            "display": "inline-flex",
            "margin": "0 5px 12px",
          }
        }
      >
        <div
          data-css-1lu7sfl=""
        >
          <div
            data-css-q4lcrt=""
          >
            <span
              data-css-1997kh1=""
            >
              Projects
            </span>
          </div>
        </div>
      </div>
      <div
        style={
          Object {
            "display": "inline-flex",
            "margin": "0 5px 12px",
          }
        }
      >
        <div
          data-css-1lu7sfl=""
        >
          <div
            data-css-q4lcrt=""
          >
            <span
              data-css-1997kh1=""
            >
              Video Courses
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots EmptyState/illustrations/large bookmark 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-11zs1jp=""
  />
  <h1
    data-css-10xrzzy=""
  >
    bookmark
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/large channel 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-11zs1jp=""
  />
  <h1
    data-css-10xrzzy=""
  >
    channel
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/large cloud 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-11zs1jp=""
  />
  <h1
    data-css-10xrzzy=""
  >
    cloud
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/large code 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-11zs1jp=""
  />
  <h1
    data-css-10xrzzy=""
  >
    code
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/large custom 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-11zs1jp=""
  >
    <svg
      aria-hidden={true}
      role="img"
      viewBox="0 0 128 128"
    >
      <path
        d="M44.99 43.356v-16.77L109.792 64 44.99 101.414v-16.77l-14.897 8.601v-58.49l14.897 8.601zm0 7.162l-8.695-5.02v37.003l8.695-5.02V50.518zm6.202 30.544v9.609L97.387 64 51.192 37.329v9.608l29.552 17.062-29.552 17.063zm0-7.162l17.147-9.901-17.147-9.9v19.8z"
        fill="currentColor"
      />
    </svg>
  </div>
  <h1
    data-css-10xrzzy=""
  >
    Custom Illustration
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/large data 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-11zs1jp=""
  />
  <h1
    data-css-10xrzzy=""
  >
    data
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/large document 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-11zs1jp=""
  />
  <h1
    data-css-10xrzzy=""
  >
    document
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/large magnify 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-11zs1jp=""
  />
  <h1
    data-css-10xrzzy=""
  >
    magnify
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/large signal 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-11zs1jp=""
  />
  <h1
    data-css-10xrzzy=""
  >
    signal
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/large video 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-11zs1jp=""
  />
  <h1
    data-css-10xrzzy=""
  >
    video
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/large zen 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-11zs1jp=""
  />
  <h1
    data-css-10xrzzy=""
  >
    zen
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/small bookmark 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  />
  <h1
    data-css-amk6wt=""
  >
    bookmark
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/small channel 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  />
  <h1
    data-css-amk6wt=""
  >
    channel
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/small cloud 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  />
  <h1
    data-css-amk6wt=""
  >
    cloud
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/small code 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  />
  <h1
    data-css-amk6wt=""
  >
    code
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/small custom 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  >
    <svg
      aria-hidden={true}
      role="img"
      viewBox="0 0 128 128"
    >
      <path
        d="M44.99 43.356v-16.77L109.792 64 44.99 101.414v-16.77l-14.897 8.601v-58.49l14.897 8.601zm0 7.162l-8.695-5.02v37.003l8.695-5.02V50.518zm6.202 30.544v9.609L97.387 64 51.192 37.329v9.608l29.552 17.062-29.552 17.063zm0-7.162l17.147-9.901-17.147-9.9v19.8z"
        fill="currentColor"
      />
    </svg>
  </div>
  <h1
    data-css-amk6wt=""
  >
    Custom Illustration
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/small data 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  />
  <h1
    data-css-amk6wt=""
  >
    data
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/small document 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  />
  <h1
    data-css-amk6wt=""
  >
    document
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/small magnify 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  />
  <h1
    data-css-amk6wt=""
  >
    magnify
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/small signal 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  />
  <h1
    data-css-amk6wt=""
  >
    signal
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/small video 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  />
  <h1
    data-css-amk6wt=""
  >
    video
  </h1>
</div>
`;

exports[`Storyshots EmptyState/illustrations/small zen 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  />
  <h1
    data-css-amk6wt=""
  >
    zen
  </h1>
</div>
`;

exports[`Storyshots EmptyState/mounting toggle show/hide 1`] = `
Array [
  <button
    onClick={[Function]}
    style={
      Object {
        "left": 20,
        "position": "fixed",
        "top": 20,
      }
    }
  >
    unmount
  </button>,
  <div
    data-css-g3x4fp=""
  >
    <div
      data-css-1gwvklx=""
    />
    <h1
      data-css-amk6wt=""
    >
      Filet mignon pork chop salami shank tail.
    </h1>
    <p
      data-css-hk53qc=""
    >
      Burgdoggen boudin shoulder, sirloin turducken landjaeger kielbasa brisket. Bacon venison meatball pork, alcatra capicola turducken. Salami shoulder strip steak filet mignon pork.
    </p>
    <div
      data-css-aelvpu=""
    >
      <button
        data-css-176v989=""
        disabled={false}
        style={Object {}}
      >
        <span
          data-css-15b13by=""
        >
          Do a Thing
        </span>
      </button>
    </div>
  </div>,
]
`;

exports[`Storyshots EmptyState/sizes large 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-11zs1jp=""
  />
  <h1
    data-css-10xrzzy=""
  >
    Filet mignon pork chop salami shank tail.
  </h1>
  <p
    data-css-hk53qc=""
  >
    Burgdoggen boudin shoulder, sirloin turducken landjaeger kielbasa brisket. Bacon venison meatball pork, alcatra capicola turducken. Salami shoulder strip steak filet mignon pork.
  </p>
  <div
    data-css-aelvpu=""
  >
    <button
      data-css-176v989=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        Do a Thing
      </span>
    </button>
  </div>
</div>
`;

exports[`Storyshots EmptyState/sizes small 1`] = `
<div
  data-css-g3x4fp=""
>
  <div
    data-css-1gwvklx=""
  />
  <h1
    data-css-amk6wt=""
  >
    Filet mignon pork chop salami shank tail.
  </h1>
  <p
    data-css-hk53qc=""
  >
    Burgdoggen boudin shoulder, sirloin turducken landjaeger kielbasa brisket. Bacon venison meatball pork, alcatra capicola turducken. Salami shoulder strip steak filet mignon pork.
  </p>
  <div
    data-css-aelvpu=""
  >
    <button
      data-css-176v989=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        Do a Thing
      </span>
    </button>
  </div>
</div>
`;
